migrator/migrator-servlet/src/main/webapp/index.jsp

```diff
@@ -67,6 +162,19 @@
                   " to the given phone number or the specified number has never been valid.</p>");
           // TODO: add link for users to file bugs
         }
+      } else if (request.getAttribute("fileError") == null && request.getAttribute("fileName") != null) {
+        out.print("<h3>+" + request.getAttribute("fileCountryCode") + " " + request.getAttribute("fileName") + " Migration Report</h3>");
```

Comment:
Maybe we could put the filename in single quotations and add "for the country code: " +  request.getAttribute("fileCountryCode") at the end?